bokeh/models/renderers.py

```diff
@@ -221,14 +221,14 @@ def __init__(self, **kwargs) -> None:
     and ranges.
     """)
 
-    selection_glyph = Either(Auto, Instance(Glyph), Null, default="auto", help="""
+    selection_glyph = Nullable(Either(Auto, Instance(Glyph), Null, default="auto"), help=""""
```

Comment:
This change is redundant. Either leave the original or add `Nulllable()` and remove `Null`. Those are synonyms and the choice between boils down to esthetics.

Reply:
I intended `Nullable(Either(Auto, Instance(Glyph)), default="auto", help="""""")`

You agree?

Reply:
I made correction in 546a424

Reply:
> the choice between boils down to esthetics.

@mattpap I would like to prefer `Nullable(...)` because doing that consistently will afford an opportunity to more easily highlight nullable properties in the documentation.

Reply:
I think will just remove `Null` to avoid confusion (or deprecate if panel already uses it).

Reply:
I think it is fine to keep it around, but just not openly encourage its use. It doesn't really Mather though, if users use it in custom extensions or whatever. I am only suggesting that as a convention *for us*, we pick one to use consistently (and I also suggest `Nullable` be that convention)